templates/macros/docs.html

```diff
@@ -49,3 +49,17 @@
         | regex_replace(pattern=`\[(?<id>.*)\}\s*(?P<url>.*)`, rep=`[$id]: $url`)
 }}
 {%- endmacro fix_links -%}
+
+
+{%- macro client_json_path(client_path) -%}
+{{config.extra.client_json_path }}{{ client_path }}
+{%- endmacro client_json_path -%}
+
+
+{%- macro boolean_to_checkmark(value) -%}
+    {% if value == true %}
+        ✓
+    {% else %}
+        ✗
```

Comment:
We need some color in this table, otherwise it's very hard to read :) Maybe green if it's implemented and red if it's not?